changelog-pre10.md

```diff
@@ -2301,20 +2301,20 @@ We expect this release to ship in the DevTools of Chrome 84.
+* A PWA [`maskable-icon`](https://developer.chrome.com/docs/lighthouse/pwa/maskable-icon-audit/) just looks better on your homescreen, so this new audit encourages you to have at least one available in your manifest ([#10370](https://github.com/GoogleChrome/lighthouse/pull/10370)).
 * `timing-budget` expands [budget assertions](https://github.com/GoogleChrome/lighthouse/blob/7e4fa55/docs/performance-budgets.md) to now be settable on all the performance metrics ([#9901](https://github.com/GoogleChrome/lighthouse/pull/9901), [#9925](https://github.com/GoogleChrome/lighthouse/pull/9925)).
-* The new [`charset`](https://web.dev/charset) audit ensures a proper character encoding for page content ([#10284](https://github.com/GoogleChrome/lighthouse/pull/10284), [#10389](https://github.com/GoogleChrome/lighthouse/pull/10389), [#10689](https://github.com/GoogleChrome/lighthouse/pull/10689)).
+* The new [`charset`](https://developer.chrome.com/docs/lighthouse/best-practices/charset/) audit ensures a proper character encoding for page content ([#10284](https://github.com/GoogleChrome/lighthouse/pull/10284), [#10389](https://github.com/GoogleChrome/lighthouse/pull/10389), [#10689](https://github.com/GoogleChrome/lighthouse/pull/10689)).
 * `image-size-responsive` checks that images have an aspect ratio and resolution that match well with how they are displayed on a page ([#10460](https://github.com/GoogleChrome/lighthouse/pull/10460)).
-* Updating to the latest version of `axe-core` has unlocked a number of new accessibility audits: [`aria-hidden-body`](https://web.dev/aria-hidden-body/), [`aria-hidden-focus`](https://web.dev/aria-hidden-focus/), [`aria-input-field-name`](https://web.dev/aria-input-field-name/), [`aria-toggle-field-name`](https://web.dev/aria-toggle-field-name/), [`duplicate-id-active`](https://web.dev/duplicate-id-active/), [`duplicate-id-aria`](https://web.dev/duplicate-id-aria/), [`form-field-multiple-labels`](https://web.dev/form-field-multiple-labels/), [`heading-order`](https://web.dev/heading-order/) ([#9798](https://github.com/GoogleChrome/lighthouse/pull/9798)).
+* Updating to the latest version of `axe-core` has unlocked a number of new accessibility audits: [`aria-hidden-body`](https://dequeuniversity.com/rules/axe/4.4/aria-hidden-body), [`aria-hidden-focus`](https://dequeuniversity.com/rules/axe/4.4/aria-hidden-focus), [`aria-input-field-name`](https://dequeuniversity.com/rules/axe/4.4/aria-input-field-name), [`aria-toggle-field-name`](https://dequeuniversity.com/rules/axe/4.4/aria-toggle-field-name), [`duplicate-id-active`](https://dequeuniversity.com/rules/axe/4.4/duplicate-id-active), [`duplicate-id-aria`](https://dequeuniversity.com/rules/axe/4.4/duplicate-id-aria), [`form-field-multiple-labels`](https://dequeuniversity.com/rules/axe/4.4/form-field-multiple-labels), [`heading-order`](https://dequeuniversity.com/rules/axe/4.4/heading-order) ([#9798](https://github.com/GoogleChrome/lighthouse/pull/9798)).
```

Comment:
These are they currently redirect too when you go to the web.dev links. I don't _know_ if they were the right ones at the time though. Again happy to revert these to avoid changing the changelog.